doc/classes/RigidBody2D.xml

```diff
@@ -176,6 +176,7 @@
 		<member name="inertia" type="float" setter="set_inertia" getter="get_inertia" default="0.0">
 			The body's moment of inertia. This is like mass, but for rotation: it determines how much torque it takes to rotate the body. The moment of inertia is usually computed automatically from the mass and the shapes, but this property allows you to set a custom value.
 			If set to [code]0[/code], inertia is automatically computed (default value).
+			[b]Note:[/b] To retrieve the automatically computed inertia, use [code]1 / PhysicsServer2D.body_get_direct_state(body).inverse_inertia[/code].
```

Comment:
Please, use a codeblock instead. Inline code tags should only be used for formatting identifiers and literals. Code samples should be submitted with a codeblock for better readability and so a C# equivalent could be added.

Reply:
Both code samples have been added. I also noticed that inertia is autocomputed somewhere after the `_ready` function, although I'm not sure exactly where, so I left that out to avoid misinformation.